src/v2/components/Validators/Table/index.jsx

```diff
@@ -112,10 +113,18 @@ const ValidatorsTable = ({separate}: {separate: boolean}) => {
         <Table>
           <TableHead className={classes.head}>
             <TableRow>
-              <TableCell align="center">Name/Moniker</TableCell>
-              <TableCell width={170}>Staked SOL</TableCell>
-              <TableCell width={170}>Commission</TableCell>
-              <TableCell width={130}>Uptime</TableCell>
+              <TableCell align="center">
+                Name/Moniker <HelpLink text="" term="" />
+              </TableCell>
+              <TableCell width={170}>
+                Staked SOL <HelpLink text="" term="" />
+              </TableCell>
+              <TableCell width={190}>
+                Commission <HelpLink text="" term="" />
+              </TableCell>
+              <TableCell width={130}>
+                Uptime <HelpLink text="" term="" />
+              </TableCell>
```

Comment:
@manuel-calavera are these just placeholders?

Reply:
yes, i will be provided with actual values later